dev-client/src/components/links/InternalLink.tsx

```diff
@@ -29,7 +29,7 @@ type InternalLinkProps = {
   url: string;
 };
 
-export default function InternalLink({label, onPress, url}: InternalLinkProps) {
+export function InternalLink({label, onPress, url}: InternalLinkProps) {
```

Comment:
Probably the true standard declaration for our other components would be `export const InternalLink = (` rather than `export function`.